rest/specs/src/main/filtered-resources/openapi/ingestapi.yaml

```diff
@@ -0,0 +1,221 @@
+# /**
+#  * Copyright (c) Connexta
+#  *
+#  * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+#  * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+#  * the License, or any later version.
+#  *
+#  * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+#  * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#  * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+#  * License is distributed along with this program and can be found at
+#  * <http://www.gnu.org/licenses/lgpl.html>.
+#  */
+openapi: 3.0.2
+
+info:
+  version: ${project.version}
+  title: ${project.name}
+  description: >
+    The Ingest API is defined to provide clients with the ability to
+    ingest products for storage and query capabilities.
+  contact:
+    name: Connexta
+  license:
+    name: GNU Lesser General Public License
+    url: 'http://www.gnu.org/licenses/lgpl.html'
+
+servers:
+  - url: ${rest.server.url}
+
+paths:
+  /ingest:
+    post:
+      summary: Ingest endpoint for products.
+      description: >
+        A system can use the Ingest endpoint to send one or more attachments
+        and a JSON description of a product.
+      operationId: ingest
+      tags:
+        - ingest
+      parameters:
+        - $ref: '#/components/parameters/Accept-Version'
+      requestBody:
+        $ref: '#/components/requestBodies/IngestRequest'
+      responses:
+        '202':
+          $ref: '#/components/responses/Accepted'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '501':
+          $ref: '#/components/responses/NotImplemented'
```

Comment:
✏️ Might want to add `default:       $ref: 'commons.yaml#/components/responses/DefaultError'` for all other possible errors